### PR TITLE
Feat pausing - split issuance and trading

### PR DESCRIPTION
**Pull request by julianmrodri**
* Splits `paused` into `tradingPaused` and `issuancePaused`
* Adapts contracts, tests, scripts, docs.

* `issuancePaused` is only used for `issueTo`. All the previous functions use `tradingPaused`

### CODE REVIEW

**Review by tbrent — COMMENTED** [review]
LGTM! Just suggesting one minor naming change

File: contracts/interfaces/IMain.sol [hunk 1]
```diff
@@ -94,9 +102,13 @@ interface IAuth is IAccessControlUpgradeable {
     // onlyRole(OWNER)
     function unfreeze() external;
 
-    function pause() external;
```
[the remark this answers is not in the document]
Comment (julianmrodri) [reply contracts/interfaces/IMain.sol:97]: Yes I thought about it. I think you are right. Will make the change.

[the remark this answers is not in the document]
Comment (julianmrodri) [reply contracts/interfaces/IMain.sol:97]: On similar note, what do you think about the error message being `paused issuance or frozen` intead of the current `frozen or issuance paused` ?

**Comment by Edgarruiz85** [comment]
> * Splits `paused` into `tradingPaused` and `issuancePaused`
> * Adapts contracts, tests, scripts, docs.
> * `issuancePaused` is only used for `issueTo`. All the previous functions use `tradingPaused`

**Pull request merged**